src/main/java/http/server/Handler.java

```diff
@@ -0,0 +1,5 @@
+package http.server;
+
+public interface Handler {
+    public void handle(Response response);
```

Comment:
Do you handle a response?

Common name, but I wonder if you can find something more descriptive/appropriate?